server/splunk/logs_test.go

```diff
@@ -15,6 +15,7 @@ var authToken = "eyJraWQiOiJzcGx1bmsuc2VjcmV0IiwiYWxnIjoiSFM1MTIiLCJ2ZXIiOiJ2MiI
 var server = "https://splunkapi.opsolutions.dev"
 
 func Test_splunk_Logs(t *testing.T) {
+	t.Skip()
```

Comment:
Is there a ticket we can link?

Reply:
@hanzei do you mean add a new ticket and add it into a comment?  I'm not sure I understand your question.

Reply:
`t.Skip()` accept a slice of arguments e.g. one string. You can link a ticket in the arguments which will then be printed as part of `go test`. This way it's obvious who will fix this test.

Reply:
Github Issues created and linked in the `Skip()` arguments